x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/use_response_actions_log_table.tsx

```diff
@@ -169,6 +171,35 @@ export const useResponseActionsLogTable = ({
           };
         });
 
+        const getOutputContent = () => {
+          if (isExpired) {
+            return OUTPUT_MESSAGES.hasExpired(command);
+          }
+
+          if (!isCompleted) {
+            return OUTPUT_MESSAGES.isPending(command);
+          }
+
+          if (!wasSuccessful) {
+            return OUTPUT_MESSAGES.hasFailed(command);
+          }
+
+          if (!isGetFileCommand) {
+            return OUTPUT_MESSAGES.wasSuccessful(command);
+          } else {
+            return (
+              <>
+                {OUTPUT_MESSAGES.wasSuccessful(command)}
+                <ResponseActionFileDownloadLink
+                  action={item}
+                  textSize="xs"
+                  data-test-subj={getTestId('getFileDownloadLink')}
+                />
+              </>
+            );
+          }
```

Comment:
only a small suggestion: `isGetFileCommand` means that it is a get-file command, but `!isGetFileCommand` does not necessarily mean success, so I think checking for file command and then going with success as default could be a bit cleaner, and also more easily extendible.

```
          if (isGetFileCommand) {
            return (
              <>
                {OUTPUT_MESSAGES.wasSuccessful(command)}
                <ResponseActionFileDownloadLink
                  action={item}
                  textSize="xs"
                  data-test-subj={getTestId('getFileDownloadLink')}
                />
              </>
            );
          }

          return OUTPUT_MESSAGES.wasSuccessful(command);
```

Reply:
Well, what you say would be right if this condition was placed arbitrarily somewhere higher up in the block. What I've done with the refactoring is mimic the ternary conditions but also exclude all negations (`¬p`) earlier so that it easier to reason what state the actions are in.  

If you look carefully we've already excluded `!wasSuccessful` at line 183, which means `wasSuccessful` evaluates to `true` from line 185 and onwards.

Reply:
39e25d74d5cb5d7031f9155ea7bd2a9402bed0cb